br/backup-and-restore-tool.md

```diff
@@ -337,7 +441,94 @@ br restore table \
+
+If you restore data from the Amazon S3 backend, instead of `local` storage, you need to specify the S3 storage path in the `storage` sub-command, and allow the BR node and the TiKV node to access to Amazon S3.
+
+Pass `SecretKey` and `AccessKey` of the account that has permission to access the S3 backend to the BR node. Here `SecretKey` and `AccessKey` are passed as environment variables. And then pass the permission to the TiKV node through BR.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+export AWS_ACCESS_KEY_ID=${AccessKey}
+export AWS_SECRET_ACCESS_KEY=${SecretKey}
+```
+
+When restoring data using BR, explicitly specify the parameters `--s3.region` and `--send-credentials-to-tikv`. `--s3.region` indicates the region where S3 is located, and `--send-credentials-to-tikv` means passing the privilege to access S3 to the TiKV node.
+
+`Bucket` and `Folder` in the `--storage` parameter represent the S3 bucket and the folder where the data to be restored is located.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+br restore full \
+    --pd "${PDIP}:2379" \
+    --storage "s3://${Bucket}/${Folder}" \
+    --s3.region "${region}" \
+    --send-credentials-to-tikv=true \
+    --log-file restorefull.log
+```
+
+### Restore incremental data
+
+Restoring incremental data is similar to [restoring full data using BR](#restore-all-the-backup-data). Note that when restoring incremental data, make sure that all the data backed up before `last backup ts` has been restored to the target cluster.
+
+### Restore Raw KV (experimental feature)
+
+> **Warning:**
+> 
+> This feature is in the experiment, without being thoroughly tested. It is highly **not recommended** to use this feature in the production environment.
+
+Similar to [backing up Rwa KV](#back-up-raw-kv-experimental-feature), you can execute the following command to restore Raw KV:
+
+{{< copyable "shell-regular" >}}
+
+br restore raw --pd $PD_ADDR \
+    -s "local://$BACKUP_DIR" \
+    --start 31 \
+    --end 3130303030303030 \
+    --format hex \
+    --cf default
+
+In the above example. all the backed up keys in the range `[0x31, 0x3130303030303030)` are restored to the TiKV cluster. The coding methods of these keys are identical to that of [keys during the backup process](#back-up-raw-kv-experimental-feature)
+
+### Online restore (experimental feature)
+
+> **Warning:**
+> 
+> This feature is in the experiment, without being thoroughly tested. It also relies on the unstable `Placement Rules` of PD. It is highly **not recommended** to use this feature in the production environment.
+
+During data restoration, writing too much data affects the performance of the online cluster. To avoid this effect as much as possible, BR supports [Placement rules](/configure-placement-rules.md) to isolate resources. In this case, downloading and importing SST are only performed on a few specified nodes (or "restore nodes" for short). Take the following steps to complete the online restore.
```

Comment:
```suggestion
During data restoration, writing too much data affects the performance of the online cluster. To avoid this effect as much as possible, BR supports [Placement rules](/configure-placement-rules.md) to isolate resources. In this case, downloading and importing SST are only performed on a few specified nodes (or "restore nodes" for short). To complete the online restore, take the following step.
```